packages/wells/src/__tests__/cogniteClient.int.spec.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Cognite AS
+
+import CogniteClient from '../cogniteClient';
+import { setupLoggedInClient } from './testUtils';
+
+describe('derived integration', () => {
+  let client: CogniteClient;
+  beforeAll(async () => {
+    client = setupLoggedInClient();
+  });
+  // test that the client behaves as stable
+  test('assets list', async () => {
+    const response = await client.assets.list();
+    expect(response.items.length).toBeGreaterThan(0);
+  });
+  test('raw get assets', async () => {
+    const response = await client.get(
+      '/api/v1/projects/subsurface-test/assets'
```

Comment:
could we make those tests optional for the developers that are not working on wells sdk?
see `packages/stable/src/__tests__/api/revisions3D.int.spec.ts`

We could check if variables are set and run only in that case.
Sot when you run tests for entire project locally and don't set wells variables it'll not fail.

(maybe some different approach, but same idea)

Reply:
Done :)